app/core/api/levels.coffee

```diff
@@ -1,6 +1,14 @@
 fetchJson = require './fetch-json'
 
 module.exports = {
-  getByOriginal: (original) ->
-    return fetchJson("/db/level/#{original}/version")
+  getByOriginal: (original, options={}) ->
+    return fetchJson("/db/level/#{original}/version", _.merge({}, options))
+  
+  fetchNextForCourse: ({ levelOriginalID, courseInstanceID, courseID, sessionID }, options={}) ->
+    if courseInstanceID
+      url = "/db/course_instance/#{courseInstanceID}/levels/#{levelOriginalID}/sessions/#{sessionID}/next"
+    else
+      # TODO: Update this route
+      url = "/db/course/#{courseID}/levels/#{levelOriginalID}/next"
+    return fetchJson(url, options)
```

Comment:
Could update route, could also transform the promise into a consistent valid response. So replace this line with something like `fetchJson(url, options).then((level) -> { level })`

But I think maybe you do have to update this route to work correctly, it's for making the teacher course guide page work?